pkg/api/admin_users.go

```diff
@@ -152,13 +152,13 @@ func AdminEnableUser(c *models.ReqContext) Response {
 
 	// External users shouldn't be disabled from API
 	authInfoQuery := &models.GetAuthInfoQuery{UserId: userID}
-	if err := bus.Dispatch(authInfoQuery); err != models.ErrUserNotFound {
+	if err := bus.Dispatch(authInfoQuery); !errors.Is(err, models.ErrUserNotFound) {
```

Comment:
Shouldn't we also be checking for `err != nil` here?

Reply:
@wbrowne I wondered about the same and I'm unsure since it was like that in the old code. You're probably right?

Reply:
@wbrowne After applying your change, tests fail so I think it's actually supposed to work this way, even if it's confusing.